HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -24,6 +24,20 @@ def esproducers_by_type(process, *types):
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+def customiseFor14282(process):
+    process.GlobalParametersRcdSource = cms.ESSource("EmptyESSource",
+        recordName = cms.string('L1TGlobalParametersRcd'),
+        iovIsRunNotTime = cms.bool(True),
+        firstValid = cms.vuint32(1))
+    return process
+
+# Add quadruplet-specific pixel track duplicate cleaning mode (PR #13753)
```

Comment:
Is fine for now: the ConfDB menus contain the ES producer, so online where this file is not used
is fine. We can clean this up later in the release with new dumps - depending what goes in
first, this should be kept for now.